tests/inputText.spec.ts

```diff
@@ -0,0 +1,21 @@
+import { chromium, test, expect } from "@playwright/test";
```

Comment:
This is a great start on our input text TDD test that we worked on using playwright, 🎭 🧪